mscore/metaedit.cpp

```diff
@@ -120,5 +135,46 @@ void MetaEditDialog::hideEvent(QHideEvent* event)
       QWidget::hideEvent(event);
       }
 
-}
+//---------------------------------------------------------
+//   openFileLocation
+//---------------------------------------------------------
+
+void MetaEditDialog::openFileLocation(const QString& path, QWidget* parent)
```

Comment:
I think this method should be in a separate class, for example `OpenFileLocation::open`
So that in the future we can see what it is and reuse it in another place where it is needed.

Reply:
done